test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -369,11 +348,21 @@ def __init__(self, out_dir, diag_path=None, server_host=None,
         # Uses Python's ternary conditional construct:
         #    a if <condition> else b
         self.diag = diag_path
-        self.outdir = os.path.abspath(out_dir)
-        self.snapshot_name = ntpath.basename(self.outdir)
+        timestamp_str = time.strftime("%Y%m%d_%H_%M_%S")
+        self.snapshot_name = "snapshot_" + timestamp_str
+
+        # Make sure that the target directory exists
+        dir_path = os.path.abspath(out_dir)
+        if os.path.isdir(dir_path) is False:
+            self.logger.error("Target directory either doesn't exist" +
+                              "or not accessible. Quitting.")
```

Comment:
The -o option is now used to specify the directory/location where the snapshot will be captured, not the snapshot directory itself. So, we check if that location exists. We only create the snapshot directory/tarball ourselves. What do you think?